sdk/scheduler/src/main/java/com/mesosphere/sdk/http/endpoints/PlansResource.java

```diff
@@ -19,6 +21,7 @@
 /**
  * API for management of Plan(s).
  */
+@Singleton
```

Comment:
whats the purpose of this?

Reply:
Fixes one of the stderr warnings, according to the warning it was defaulting to singleton anyway.